main.rb

```diff
@@ -0,0 +1,134 @@
+require_relative './book'
+require_relative './students'
+require_relative './teacher'
+require_relative './rental'
+class App
+  def initialize()
+    @start = 'Please choose an option by entering a number
+    1 - List all books
+    2 - List all people
+    3 - Create a person
+    4 - Create a book
+    5 - Create a rental
+    6 - List all rentals for a given person id
+    7 - exit'
```

Comment:
* I suggest you move all these options into a different method as in the project requirement your entry point should be in a method called `main`  [Refer back to the requirements](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_ui.md#:~:text=In%20your%20main,quit%20the%20app.)

Reply:
* The `main method` should house the following options, please check the screenshot below ⬇️.  when i start the app the main method should be your entry point 
![image](https://user-images.githubusercontent.com/52497246/137371918-8b13f08a-c074-42cd-83de-5853b049df22.png)